Docs/troubleshooting.md

```diff
@@ -0,0 +1,236 @@
+
+
+#Troubleshooting
```

Comment:
The troubleshooting guide now contains an incomplete/outdated version of the former Quick Start guide which is now missing. The "Cache Clearing" and "Monitoring and Logging" sections might be a good fit for a troubleshooting document, but the other sections "LoRa Device Provisioning", "Decoders", "Customizing", "C2D Messages" and "Mac Commands" might make more sense in the former quick start guide or a new "Advanced Features" guide.

Reply:
Yep, some are advanced feature that are not part of the troubleshooting

Reply:
I am talking to Sasch on this today.

Reply:
resolved this with Sascha, we agree on having a supplement reference